### DISCUSSION TITLE
Plan to apply setup-node cache option to all Octokit repositories

### DISCUSSION THREAD

**Discussion by oscard0m** [octokit/octokit.js#2125] [Ideas] [answered]
I would like to discuss a plan to apply `cache` option to GitHub Actions using `setup-node`.

I already created [an Octoherd script](https://github.com/oscard0m/octoherd-script-add-cache-to-node-github-action) for this but before running it in the company (I already did but closed all those PR's until this discussion is closed, sorry for all the noise caused), I would like to discuss the plan, how to assure it works, etc.

**Comment by gr2m** [comment, marked as answer]
I think we do use [`actions/cache`](https://github.com/actions/cache) in some repositories, we should remove that from the steps now and instead add the `cache: npm` option to `actions/setup-node`. Other than that the script should make sure to set `actions/setup-node@v2` if the repo is still using `actions/setup-node@v1`, but I think it is doing that already?

**Reply by oscard0m** [reply]
- 👍🏽  I have pending to create the logic to get rid of the `actions/cache` step for those GitHub Actions using it. 
- Yes, it's already bumping v1 to v2 (not considering other cases like `@v2-beta`, `@commit-hash`, etc)

**Reply by oscard0m** [reply]
I think all repos have been migrated to `cache` (as far as I know). Can I mark this discussion as answered?

**Reply by gr2m** [reply]
Yes, thank you! I'll review and merge the remaining open PRs!
